consensus/istanbul/validator.go

```diff
@@ -71,6 +71,9 @@ type Validator interface {
 	// Serialize returns binary reprenstation of the Validator
 	// can be use used to instantiate a validator with DeserializeValidator()
 	Serialize() ([]byte, error)
+
+	// AsData returns Validator representation as ValidatorData
+	AsData() *ValidatorData
```

Comment:
I don't think we should be adding functions to these interfaces unless it accomplishes something that we can't do otherwise. These are already too bloated for the intended uses of interfaces and we really should add to that.

Reply:
I've removed the `AsData()` from the ValidatorSet inteface... but for `Validator` is need to map Validators to Data